app/Resources/views/macros/forms.html.twig

```diff
@@ -81,7 +81,7 @@
             <span class="text-muted">({{ msg("optional") }})</span>
         {% endif %}
     </label>
-    <input type="date" class="form-control" name="{{ type }}" placeholder="2016-11-01"{% if required %} required{% endif %} />
+    <input type="date" class="form-control" name="{{ type }}" placeholder="YYYY-MM-DD"{% if required %} required{% endif %} />
```

Comment:
No, I mean the dates the controller uses if no date is given.  For example: https://github.com/x-tools/xtools-rebirth/blob/autoedits/src/AppBundle/Controller/AutomatedEditsController.php#L125